core/src/main/scala/org/elasticmq/actor/queue/QueuePersister.scala

```diff
@@ -0,0 +1,11 @@
+package org.elasticmq.actor.queue
+
+import org.elasticmq.QueueData
+
+trait QueuePersister {
+
+  def persist(queue: QueueData): Unit
```

Comment:
these should probably allow async operations, that is return `Future[Unit]` (+ proper integration at the call site). 

I wonder what should happen if persisting fails ... probably propagating the exception is the best we can do